fix(css): update path() and shape-outside pages (#34961)
mdn/content: files/en-us/web/css/basic-shape/path/index.md

- [`<fill-rule>`](/en-US/docs/Web/SVG/Attribute/fill-rule) {{optional_inline}}

>   - : Defines what parts of the shape to be considered inside the shape. The possible values include:

Reviewer: ```suggestion
  - : Defines what parts of the path are inside the shape. The possible values include:
```

    > **Warning:** `<fill-rule>` is not supported in {{cssxref("offset-path")}} and using it invalidates the property.

- {{cssxref("string")}}

>   - : A [data string](/en-US/docs/Web/SVG/Attribute/d), encompassed in quotes, for defining an [SVG path](/en-US/docs/Web/SVG/Element/path). The SVG path data string contains [path commands](/en-US/docs/Web/SVG/Attribute/d#path_commands) that implicitly use pixel unit. An empty path is considered invalid.

Reviewer: ```suggestion
  - : A [data string](/en-US/docs/Web/SVG/Attribute/d), contained in quotes, which defines an [SVG path](/en-US/docs/Web/SVG/Element/path). The SVG path data string contains [path commands](/en-US/docs/Web/SVG/Attribute/d#path_commands) that implicitly use pixel units. An empty path is considered invalid.
```

path(evenodd,"M 10 80 C 40 10, 65 10, 95 80 S 150 150, 180 80");
```

> ### Use as an offset-path value

Reviewer: ```suggestion
### Using a `path()` function as an `offset-path` value
```

> The `path()` function has been used in the following example to create an elliptical path for the ball to move on.

Reviewer: ```suggestion
A `path()` function has been provided as an {{cssxref("offset-path")}} value in the following example to create an elliptical path for a ball to move along.
```
